src/librustdoc/html/static/search.js

```diff
@@ -1012,7 +1036,7 @@ window.initSearch = function(rawSearchIndex) {
                 "href=\"https://doc.rust-lang.org/book/index.html\">Rust Book</a> for " +
                 "introductions to language features and the language itself.</li><li><a " +
                 "href=\"https://docs.rs\">Docs.rs</a> for documentation of crates released on" +
-                " <a href=\"https://crates.io/\">crates.io</a>.</li></ul></div>";
+                " <a href=\"https://crates.io/\">crates.io</a>.</li></ul>";
```

Comment:
Do we support browsers that don't handle [template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)?

Reply:
I'm not sure, needs to be checked. From the page, only IE doesn't seem to support and we don't support IE anymore either, so just need to check for other browsers too. Let's check that in another PR though.

Reply:
I'd also be interested in trying out the HTML `<template>` element: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template

Seems well supported: https://caniuse.com/?search=template

If we're going for the quick fix (which I think is reasonable), I would rather do a small tweak to the existing string-generation code than reorganize this code path completely. Can you explain more in the PR description under what conditions the spurious `<div>~ was generated?

Reply:
This `<div>` wasn't "too much", it's simply that now we generate it using JS and then we add the HTML into it, so I had to remove the extra closing tag since it's now unneeded.